_includes/pbc/all/install-features/202400.0/install-the-warehouse-user-management-feature.md

```diff
@@ -65,8 +65,9 @@ Make sure that the following changes have been triggered in transfer objects:
 | WarehouseUserAssignmentCollectionRequest        | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentCollectionRequest        |
 | WarehouseUserAssignmentCollectionResponse       | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentCollectionResponse       |
 | WarehouseUserAssignmentCollectionDeleteCriteria | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentCollectionDeleteCriteria |
-| ApiWarehouseUserAssignmentsAttributes           | class    | created    | src/Generated/Shared/Transfer/ApiWarehouseUserAssignmentsAttributesTransfer   |
-| ApiWarehousesAttributes                         | class    | created    | src/Generated/Shared/Transfer/ApiWarehousesAttributes                         |
+| WarehouseUserAssignmentsBackendApiAttributes    | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentsBackendApiAttributes    |
+| WarehousesBackendApiAttributes                  | class    | created    | src/Generated/Shared/Transfer/WarehousesBackendApiAttributes                  |
+| UsersBackendApiAttributes                       | class    | created    | src/Generated/Shared/Transfer/UsersBackendApiAttributes                       |
```

Comment:
The same question for `UsersBackendApiAttributes`, it's part of `Spryker Core Back Office` feature. Is it somewhere defined in the warehouse user connector?

Reply:
Didn't found in related modules. So UsersBackendApiAttributes transfer was removed.